server/routes/users.js

```diff
@@ -1,15 +1,58 @@
 var express = require('express');
 var router = express.Router();
+const jwt = require('jsonwebtoken');
+const dotenv = require('dotenv');
 
 const { PrismaClient } = require('@prisma/client')
 
+dotenv.config();
+
 const prisma = new PrismaClient()
 
 /* GET users listing. */
 router.get('/', async function(req, res, next) {
   const allMyUsers = await prisma.user.findMany()
-  console.log(allMyUsers);
   res.send(allMyUsers)
 });
 
+async function tokenify (user) {
+
+  
+  let jwtSecretKey = process.env.JWT_SECRET_KEY;
+  const token = jwt.sign(user, jwtSecretKey, {expiresIn: '100s'});
+  return token;
+
+}
+
+async function getUserByEmail (email, password) {
```

Comment:
I can write a test for this endpoint while I'm working on fixing up testing & CI